### ISSUE TITLE
SW360 can connect some folder/server of fossology  

### ISSUE THREAD

**Issue by KoukiHama** [eclipse/sw360#1167]
[//]: # (Copyright Siemens AG, 2021. Part of the SW360 Portal Project)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

### Description

One SW360 instance should connect multi fossology instance.

Refer to this post.
_Originally posted by @mcjaeger in https://github.com/eclipse/sw360/issues/857#issuecomment-809731853_

#### Steps followed and expected result

User would choose instance/folder of fossology.

**Comment by mcjaeger** [comment]
I agree that users shall have the ability to put their own fossology server or the same fossology server with a different folder. therefore, the sw360 should be able to maintain different fossology settings at the same time.
